scripts/percona-release.sh

```diff
@@ -322,38 +322,50 @@ function check_setup_command {
 }
 #
 if [[ ${COMMANDS} != *${1}* ]]; then
-  echo "ERROR: Unknown action specified: ${1}"
-  show_help
-  exit 2
+  if [ "$1" != "-y" ]; then
```

Comment:
it should be available only for _setup_
there is function check_setup_command. please think about it

Reply:
Gotcha, thanks for clarification :)

Done!